tests/commands/record/test_tests.py

```diff
@@ -37,7 +37,7 @@ def remove_backslash(input: str) -> str:
 
         # normal.xml
         self.assertIn('open_class_user_test.rb', gzip.decompress(
-            responses.calls[2].request.body).decode())
+            responses.calls[3].request.body).decode())
```

Comment:
updating response index for the `record tests` got an initial verification request.
same for the rest